src/catkin_pkg/package.py

```diff
@@ -698,6 +698,14 @@ def parse_package_string(data, filename=None, warnings=None):
     unknown_tags = set([n.tagName for n in nodes if n.tagName not in known.keys()])
     if unknown_tags:
         errors.append('The manifest (with format version %d) must not contain the following tags: %s' % (pkg.package_format, ', '.join(unknown_tags)))
+    if 'run_depend' in unknown_tags and pkg.package_format >= 2:
+        errors.append('The manifest of package "%s" (with format version %d) '
+                      'does not support <run_depend>.' % (pkg.name, pkg.package_format))
+        errors.append('Please replace <run_depend> tags with <exec_depend> tags.')
+    elif 'exec_depend' in unknown_tags and pkg.package_format < 2:
+        errors.append('The manifest of package "%s" (with format version %d) '
+                      'does not support <exec_depend>.' % (pkg.name, pkg.package_format))
+        errors.append('Either update to a newer format or replace <exec_depend> tags.')
```

Comment:
> ... replace <exec_depend> tags.

I would rather include the information what to replace it with: `... replace <exec_depend> tags with <run_depend> tags`

Reply:
Thanks. changed.